src/main/webapp/resources/js/pages/analysis/components/jobErrors/StandardOutput.jsx

```diff
@@ -6,13 +6,15 @@
  */
 
 import React from "react";
-import { CodeBlock } from "../../../../components/typography";
+import { OutputWrapper } from "../../../../components/OutputFiles/OutputWrapper";
 
 export function StandardOutput({ galaxyJobErrors, currIndex }) {
   // Returns the standard output for the given index from the jobErrors object
   function getStandardOutput(index = 0) {
     return (
-      <CodeBlock>{galaxyJobErrors[index].standardOutput.trim()}</CodeBlock>
+      <OutputWrapper overflowRequired={true} style={{ whiteSpace: "pre-wrap" }}>
```

Comment:
Interesting since you have it here, why is it on the others?

Reply:
Updated in 02cc213ba4a9097b9f3be5383bd1a1a5f5973211